test/extensions/filters/http/compressor/compressor_filter_speed_test.cc

```diff
@@ -71,6 +71,33 @@ class MockZstdCompressorFactory : public Envoy::Compression::Compressor::Compres
   const uint64_t chunk_size_{4096};
 };
 
+class MockBrotliCompressorFactory : public Envoy::Compression::Compressor::CompressorFactory {
+public:
+  MockBrotliCompressorFactory(uint32_t quality) : quality_(quality) {}
+
+  Envoy::Compression::Compressor::CompressorPtr createCompressor() override {
+    return std::make_unique<Compression::Brotli::Compressor::BrotliCompressorImpl>(
+        quality_, window_bits_, input_block_bits_, disable_literal_context_modeling_, mode_,
+        chunk_size_);
+  }
+
+  const std::string& statsPrefix() const override {
+    CONSTRUCT_ON_FIRST_USE(std::string, "brotli.");
+  }
+  const std::string& contentEncoding() const override {
+    return Http::CustomHeaders::get().ContentEncodingValues.Brotli;
+  }
+
+private:
+  const uint64_t chunk_size_{4096};
```

Comment:
would it make sense to parameterize your performance tests over some of these values? chunk size in particular?

Reply:
Brotli has its default values for params in `config.cc`, but they are only able to used in this file.

Reply:
OK, thanks!

Reply:
So this still needs to be resolved; the 4096 chunk size constant should only be in one place. Same with the other constants below (24, and 18).